docs/source/index.rst

```diff
@@ -55,9 +55,88 @@ rules and transform your objects with as little boilerplate as possible.
   // CoffeeMade(24, "Espresso", "Piotr", "2020-02-03T20:26:59.659647+07:00[Asia/Bangkok]")
 
 
-Read :ref:`transformers/getting-started:Getting started with transformers` to learn more about
+Read :ref:`partial-transformers/getting-started:Getting started with transformers` to learn more about
 Chimney's transformers.
 
+Partial transformers
```

Comment:
Seems we have too many, too detailed code examples in this section.

That shouldn't be in-depth explanation what you can do with partial transformers (we have dedicated page for that!), but rather introductory _catch-the-idea_ example that signals that we support transformers that can fail.

The same simple example should be used in `README.md`, I guess?